flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/Table.java

```diff
@@ -111,7 +113,15 @@
 	 *
 	 * <pre>
 	 * {@code
-	 *   tab.select('key, 'value.avg + " The average" as 'average)
+	 *   tab.select($("key"), $("value").avg(), lit("The average").as("average"))
```

Comment:
should be `plus()`

Reply:
I thought this was a mistake of the original expression. Did not know that you can actually use `+` for strings.